keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListener.kt

```diff
@@ -146,59 +133,14 @@ class ArtifactListener(
   private fun getLatestStoredVersion(artifact: DeliveryArtifact): String? =
     repository.artifactVersions(artifact).sortedWith(artifact.versioningStrategy.comparator).firstOrNull()
 
-  private suspend fun getLatestDeb(artifact: DebianArtifact): String? =
```

Comment:
All the code from here on down has been moved to `DebianArtifactPublisher` and `DockerArtifactPublisher`.